### PR TITLE
Update buck_setup.sh

### PR DESCRIPTION
**Pull request by linbinyu**
Add a parameter for proxy setup when running this script in devserver

### CODE REVIEW

**Review by malfet — COMMENTED** [review]
Can you please do something like $PROXY_CONFIG=$1 and then use this variable rather than $1

File: scripts/buck_setup.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+$PROXY_CONFIG = $1
```
Comment (malfet) [inline scripts/buck_setup.sh:2]: Isn't it how it supposed to be?
```suggestion
PROXY_CONFIG = "$1"
```

**Comment by linbinyu** [comment]
tested on devserver

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**